engine/storage/image/src/org/apache/cloudstack/storage/image/store/TemplateObject.java

```diff
@@ -28,7 +28,6 @@
 import org.apache.cloudstack.engine.subsystem.api.storage.DataObjectInStore;
 import org.apache.cloudstack.engine.subsystem.api.storage.DataStore;
 import org.apache.cloudstack.engine.subsystem.api.storage.ObjectInDataStoreStateMachine;
-import org.apache.cloudstack.engine.subsystem.api.storage.ObjectInDataStoreStateMachine.Event;
```

Comment:
It's not a compilation error but removing this import cause a variable definition to use the whole import path.